javascript/builtins/RegExp.json

```
@@ -426,6 +426,57 @@
             }
           }
         },
+        "hasIndices": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/RegExp/hasIndices",
+            "support": {
+              "chrome": {
+                "version_added": "91"
+              },
+              "chrome_android": {
+                "version_added": "91"
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": "88"
+              },
+              "firefox_android": {
+                "version_added": "88"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
```

No reason to expect WebView to omit this:

```suggestion
                "version_added": "90"
```

---

Done.